gradle.properties

```diff
@@ -1 +1 @@
-version=4.1.1
+version=4.1.2
```

Comment:
In the aws-cloudwatch and azure-cluster-discovery extension you bumped the minor version because of the aws/azure-sdk bumps, why is this here different?